deployment/makefiles/config-emr.mk

```diff
@@ -0,0 +1,20 @@
+export NAME := OSM Vector Tile Ingest - ${USER}
+export USE_SPOT := true
+export YARN_OVERHEAD := 1500
+
+export MASTER_INSTANCE:=m3.2xlarge
+export MASTER_PRICE := 0.13
+export DRIVER_MEMORY := 20000M
+export DRIVER_CORES := 8
+
+export WORKER_INSTANCE:=m3.xlarge
+export WORKER_PRICE := 0.13
+export WORKER_COUNT := 200
+export EXECUTOR_MEMORY := 21500M
+export EXECUTOR_CORES := 8
```

Comment:
Have you seen a benefit to providing these vs. allowing EMR to set these automatically?

Reply:
These are in the configs, but you can see that I don't use them in the actual add-steps command in the Makefile.  I'm relying on the automatic allocation stuff now.  I can clean up these configs to clear up confusion later.